hopbox/hopbox_app/templates/home.html

```diff
@@ -7,52 +7,53 @@
     <title>HopBox Subscription Service</title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href=static/style.css>
     {% load static %}
+    <link rel="stylesheet" href=static/style.css>
 </head>
 <body>
     <div id="container">
         <nav class="navbar navbar-expand-lg navbar-white  bg-white mb-3" style="border-bottom: 3px solid #91B957;">
             <div class="container-fluid">
                 <a class="navbar-brand ms-2 fw-bold active" href="/userdashboard">
-                    <img src="{% static 'images/HopBox-logo.png' %}" alt="logo_placeholder" width="340" height="155" class="d-inline-block align-text-top rounded">
+                    {% load static %} <a href="/"> <img class='logo' src="{% static 'images/hopbox-logo-lg.png' %}" alt="HopBox-Logo" /></a>
+                    <!-- <img src="HopBox-logo.png" alt="logo_placeholder" width="340" height="155" class="d-inline-block align-text-top rounded"> -->
                 </a>
                 <p class="mt-2 fs-5">   
                     <a href="/cart" class="me-4 text-blue">Cart</a>|
-                    <a href="/manage-account" class="me-4 text-blue">Manage Account</a>|
+                    <a href="/edit_account" class="me-4 text-blue">Manage Account</a>|
```

Comment:
did you change the url path in urls.py to match this? I had originally set the path to `/manage-account` because that's what I had written in the urls file